src/.vuepress/config.js

```diff
@@ -63,9 +63,9 @@ module.exports = {
         children: ['/build/tbcplaceholder.md', '/build/hardware.md'],
       },
       {
-        title: 'Funding & Earning with CUDOS Tokens',
+        title: 'Common Tasks with Keplr Wallet and Cudos Explorer',
```

Comment:
Replace with 'Using CUDOS' ? I think this is too wordy currently

Reply:
I was originally going to name it 'Working with your CUDOS Tokens', but then decided to include keplr and explorer.. how about that?

Reply:
Maybe somewhere in between like 'Using CUDOS Tokens'? as not everyone reading this will own the token yet